app.js

```diff
@@ -1,89 +1,62 @@
-var express = require('express');
-var path = require('path');
-var favicon = require('serve-favicon');
-var logger = require('morgan');
-var cookieParser = require('cookie-parser');
-var bodyParser = require('body-parser');
-var WebSocketServer = require('ws').Server
-  , wss = new WebSocketServer({ port: 8080 });
+var fs = require('fs');
+var os = require('os');
+var WebSocketServer = require('ws').Server,
+    wss = new WebSocketServer({ port: 8080 });
+var databaseDir = "../kegberrydb";
+var userDbFile = databaseDir + "/users.nosql";
+var pourDbFile = databaseDir + "/pours.nosql";
+
+// Set up databases if they don't exist
+if (!fs.existsSync(databaseDir)){
+    fs.mkdirSync(databaseDir);
+}
+if (!fs.existsSync(userDbFile)){
+    fs.writeFileSync(userDbFile, "{\"id\":\"default_user\",\"name\":\"Default User\"}" + os.EOL);
+}
+if (!fs.existsSync(pourDbFile)){
+    fs.writeFileSync(pourDbFile, "{}" + os.EOL);
+}
+
+var users = require('./keg/users.js')(userDbFile);
+var pours = require('./keg/pours.js')(pourDbFile);
 
 // Spawn an app for temperature reading
+var TEMP_POLLING_SEC = 10;
 var spawn = require('child_process').spawn,
-    py    = spawn('python', ['node_modules/Adafruit_Python_DHT-master/examples/get_temperature.py']);
+    py    = spawn('python', ['./keg/get_temperature.py', TEMP_POLLING_SEC]);
 
-var routes = require('./routes/index');
-var users = require('./routes/users');
+py.on('error', (err) => {
+  console.log("Couldn't spawn temperature polling. Make sure python is installed.")
+});
 
-var app = express();
+var MOCK_POURS = false;
 
-// view engine setup
-// app.set('views', path.join(__dirname, 'views'));
-// app.set('view engine', 'jade');
+var flowmeter = require('./keg/flowmeter')({
+	pin: 40,
+	tickCalibration: 0.00089711713,
+	timeBetweenPours: 3000, 
```

Comment:
I think these are self-explanatory named parameters. won't put in variables since they're not used anywhere else in the file